modules/instance_template/main.tf

```diff
@@ -69,6 +69,7 @@ resource "google_compute_instance_template" "tpl" {
   metadata_startup_script = var.startup_script
   region                  = var.region
   min_cpu_platform        = var.min_cpu_platform
+  resource_policies       = var.resource_policies
```

Comment:
Looks like this was added in [TPG 4.53](https://github.com/hashicorp/terraform-provider-google/blob/3edc574a3a8faf3cb3f4fe179cfffd7b3ba5276f/CHANGELOG.md#4530-february-13-2023). Could we also bump the min version here https://github.com/terraform-google-modules/terraform-google-vm/blob/1c8f7759262c0a24f83268fcb7561bb7e01e7d8b/modules/instance_template/versions.tf#L20

Reply:
Great catch! I updated the version constraint 👍